tensorflow/lite/micro/tools/make/helper_functions.inc

```diff
@@ -475,13 +475,14 @@ endef
 # test and non-test binaries.
 
 # Files that end with _test are added as test targets (i.e. can be executed with
-# make test_<target>. ALl others can be executed with make run_<target>
+# make test_<target>. All others can be executed with make run_<target>
 #
 # Arguments are:
 # 1 - Name of target.
 # 2 - C/C++ source files
 # 3 - C/C++ header files
-# 4 - if "exclude", then the non-test target will be excluded from
+# 4 - Model sources and model test inputs in .tflite or .bmp format.
+# 5 - if "exclude", then the non-test target will be excluded from
 #     MICROLITE_BUILD_TARGETS. This exception is needed because not all the
 #     microlite_test targets (e.g. the examples) are buildable on all platforms.
```

Comment:
I don't see anywhere in the code that we are actually using "exclude". Might be worth removing it completely. Can be a separate PR.

Reply:
Removed it in this PR since we are already messing with arguments.